plans/benchmarks/benchmarks.go

```diff
@@ -177,15 +159,12 @@ func BarrierBench(runenv *runtime.RunEnv) error {
 			if err != nil {
 				return err
 			}
-			<-client.MustBarrier(ctx, sync.State(testState), testInstanceNum).C
+			<-client.MustBarrier(ctx, testState, testInstanceNum).C
 
-			duration := time.Since(barrierTestStart)
-			emitTime(runenv, tst.Name, duration)
+			elapsed := time.Since(barrierTestStart)
```

Comment:
Why timer and a point including the elapsed time?

Reply:
Nevermind, the timer keeps an updated accumulation of the elapsed time. Makes sens.